.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+      - release/v*
```

Comment:
I don't think we want to add a new release asset on every push we do to a release branch, right?

Reply:
Made it so that it will be triggered when the release is made.